### PR TITLE
Finished grid component

### PR DESCRIPTION
**Pull request by Naman-Saxena1**
Finished Grid

### CODE REVIEW

File: Components/Grid/grid.html [hunk 1]
```diff
@@ -76,8 +76,126 @@
...
                 <p>
-                    Grids will come here.
+                    Grids are used to build responsive layouts in grid style fashion (Columns and Rows).
                 </p>
+                <br>
+                <br>
+                <br>
+                <br>
+                <h2>Grid of 2 Items with 50-50 ratio</h2>
+                <div class="grid-container">
```
Comment (omkarpatke) [inline Components/Grid/grid.html:86]: Why are you using two divs instead of apending a class on grid1
like for example "grid-container grid-1"

**Review by omkarpatke — COMMENTED** [review]
I see only one suggestion for you otherwise it looks good to merge. btw your UI is very good.

**Comment by varun-Tad** [comment]
Looks good for me to merge. Have to agree your UI looks good.

File: Components/Grid/grid.html (hunk 1 above)
Comment (Naman-Saxena1) [reply Components/Grid/grid.html:86]: Thanks for suggestion, your point makes sense, I will work on it in next PR.

**Comment by Naman-Saxena1** [comment]
> I see only one suggestion for you otherwise it looks good to merge. btw your UI is very good.

Thanks! 😄

**Comment by Naman-Saxena1** [comment]
> Looks good for me to merge. Have to agree your UI looks good.

Thanks! 😄

**Pull request merged**